polar-core/src/warnings.rs

```diff
@@ -5,6 +5,42 @@ use super::terms::*;
 
 use std::collections::{hash_map::Entry, HashMap};
 
+pub fn common_misspellings(t: &str) -> Option<String> {
+    let misspelled_type = match t {
+        "int" => Some("Number"),
+        "Integer" => Some("Number"),
+        "i32" => Some("Number"),
+        "i64" => Some("Number"),
+        "u32" => Some("Number"),
+        "u64" => Some("Number"),
+        "size_t" => Some("Number"),
+        "usize" => Some("Number"),
+        "float" => Some("Number"),
+        "Float" => Some("Number"),
```

Comment:
Wait, we do register `Float` and `Integer` as separate types right?

Reply:
yeah these should be `Float` or `Integer`; not `Number`